pkg/server/helpers.go

```diff
@@ -407,3 +409,18 @@ func getPodCNILabels(id string, config *runtime.PodSandboxConfig) map[string]str
 		"IgnoreUnknown":              "1",
 	}
 }
+
+// getRuntime returns the runtime configuration
+// If the container is privileged, it will return
```

Comment:
If the workload is trusted, it will return the privileged runtime else not.

I'll take care of this.